nf_core/modules/bump_versions.py

```diff
@@ -0,0 +1,322 @@
+"""
+Bump versions for all modules on nf-core/modules
+or for a single module
+"""
+
+
+from __future__ import print_function
+import logging
+import questionary
+import re
+from requests.models import requote_uri
+import rich
+from rich.console import Console
+from rich.table import Table
+from rich.markdown import Markdown
+import rich
+from nf_core.utils import rich_force_colors
+from nf_core.lint.pipeline_todos import pipeline_todos
+import sys
+
+import nf_core.utils
+import nf_core.modules.module_utils
+from nf_core.modules.pipeline_modules import ModulesRepo
+from nf_core.modules.nfcore_module import NFCoreModule
+
+log = logging.getLogger(__name__)
+
+
+class ModuleVersionBumper(object):
+    def __init__(
+        self,
+        dir=".",
+    ):
+        self.dir = dir
+
+    def bump_versions(self, module=None, all_modules=False, show_uptodate=False):
+        """
+        Bump the container and conda version of single module or all modules
+
+        Looks for a bioconda tool version in the `main.nf` file of the module and checks whether
+        are more recent version is available. If yes, then tries to get docker/singularity
+        container links and replace the bioconda version and the container links in the main.nf file
+        of the repsective module.
+
+        Args:
+            module: a specific module to update
+            all_modules: whether to bump versions for all modules
+        """
+        self.up_to_date = []
+        self.updated = []
+        self.failed = []
+        self.show_up_to_date = show_uptodate
+
+        # Verify that this is not a pipeline
+        repo_type = nf_core.modules.module_utils.get_repo_type(self.dir)
+        if not repo_type == "modules":
+            log.error("This command only works on the nf-core/modules repository, not on pipelines!")
+            sys.exit()
```

Comment:
```suggestion
            return False
```
Or perhaps replace the log message with `ModuleException(...)`.